src/bookshelf/mapper.ts

```diff
@@ -34,7 +34,7 @@ export default class Bookshelf implements I.Mapper {
    * @param bookshelfOptions
    * @param template
    */
-  mapRelations(model: Model, type: string, bookshelfOptions: I.BookshelfOptions = {relations: true}, template?: ISerializerOptions): void {
+  mapRelations(model: Model, type: string, bookshelfOptions: I.BookshelfOptions = {relations: true}, template?: ISerializerOptions, excludeIdAndTypeColumns = false): void {
```

Comment:
I'll just add it here, but it applies to a lot of places: Many times we want to customize the behavior of the mapper (we don't want links, omit/rename relationships, allow id attrs, etc).

However, if options are being received as parameters we have the problem of their order and optionality:
```
-- Was that the 3rd argument or the 2nd?
-- Oh wait, I have to put undefined here because I'm not changing this.
-- Oh damn, they updated and added a new option 
   and I have to put a new undefined in all my function calls.
```

Then, setting the options in a single bag argument is a reasonable approach to solve this. In other words, if you want to contribute with a new behavior option, add it to the option bag (the ``BookshelfOptions` interface), not as an extra argument on so many function calls ;)